### PR TITLE
en: add how to delete backup cr

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

### What is changed, added or deleted? (Required)

- Add a section about how to delete the backup CR (https://github.com/pingcap/docs-tidb-operator/pull/566, https://github.com/pingcap/docs-tidb-operator/pull/591)
- Add a step in deploy-on-aws-eks.md (https://github.com/pingcap/docs-tidb-operator/pull/523)

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.1 (TiDB Operator 1.1 versions)
- [ ] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-tidb-operator/pull/566, https://github.com/pingcap/docs-tidb-operator/pull/591, https://github.com/pingcap/docs-tidb-operator/pull/523
- Other reference link(s):

### CODE REVIEW

**Comment by cofyc** [comment]
@AndreMouche it seems that the promotion of @lichunzhu still does not take effect

**Comment by cofyc** [comment]
> @AndreMouche it seems that the promotion of @lichunzhu still does not take effect

oh, I forgot that this repo belongs to `sig-docs`. only reviewers of `sig-docs` can LGTM

**Comment by ran-huang** [comment]
> > @AndreMouche it seems that the promotion of @lichunzhu still does not take effect
> 
> oh, I forgot that this repo belongs to `sig-docs`. only reviewers of `sig-docs` can LGTM

Actually, @lichunzhu already qualifies as a reviewer. I'll update the member list and add him as a review of Docs SIG 😁

File: en/deploy-on-aws-eks.md [hunk 1]
```diff
@@ -284,6 +284,30 @@ You can use the `terraform output` command to get the output again.
...
 
+4. View the startup status of the TiDB cluster:
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl --kubeconfig credentials/kubeconfig_${eks_name} get pods -n ${namespace}
+    ```
+
+    When all the Pods are in the `Running` and `Ready` states, the TiDB cluster is successfully started. For example:
```
Comment (lilin90) [inline en/deploy-on-aws-eks.md:295]: It seems that "or" is more appropriate?
```suggestion
    When all the Pods are in the `Running` or `Ready` state, the TiDB cluster is successfully started. For example:
```

**Pull request merged**